tests/cli/snapshots/snap_test_schema_generation.py

```diff
@@ -259,7 +259,7 @@
                     "enum": [
                         "sub-pipeline-component-correct-docstr"
                     ],
-                    "title": "Parameter description looks correct and it is not included in the class description, terminates here",
```

Comment:
Could you keep this check somehow? It is not automated enough, but I'd prefer to have it in some form at least.

[Related change](https://github.com/bakdata/kpops/pull/309/files#diff-f98effd3c9895a821b65f6c97e709d3e9114cf52d41c454c5a87364187fe35dfL52-L59)

Reply:
you're right, good thing you noticed. I've added an example attribute back to keep this check